.github/workflows/build.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           if (Test-Path C:\ProgramData\docker\config\daemon.json) {
             $config=(Get-Content C:\ProgramData\docker\config\daemon.json | ConvertFrom-json)
           }
-          $config."insecure-registries" = @("$IPAddress/32")
+          $config | Add-Member -Force -Name "insecure-registries" -value @("$IPAddress/32") -MemberType NoteProperty
```

Comment:
Thank you for doing this! I had tried, and then dropped the ball with this: https://github.com/buildpacks/pack/pull/1522